App/Sagas/TextileSagas.js

```diff
@@ -264,12 +264,36 @@ export function * photosTask () {
       return true
     })
 
+
+    let photoUploads = []
```

Comment:
This change looks familiar... Did this all just come from merging in our previous work around triggering a single UI update for all added photos?

Reply:
The target of this MR is your WIP, maybe it wasn't up to date with master yet?